tests/api/driver_core.c

```diff
@@ -11,29 +11,34 @@
 #include "tests/test_define.h"
 
 static ta_core_t ta_core;
-
+#define txn_num_in_bundle 2
```

Comment:
Is the number of transactions in a bundle always 2?

Reply:
For this moment, we only place 2 transactions in a bundle to ensure all the transactions can be returened

Reply:
Letters in the macro should be upper case.